src/main/java/tests/OperationStrategyTest.java

```diff
@@ -0,0 +1,25 @@
+package tests;
+
+import core.basesyntax.*;
+import org.junit.jupiter.api.Test;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import static org.junit.jupiter.api.Assertions.assertInstanceOf;
+
+public class OperationStrategyTest {
+    @Test
+    public void testOperationStrategy() {
+        Map<Operation, OperationHandler> operationHandlers = new HashMap<>();
+        operationHandlers.put(Operation.BALANCE, new BalanceOperation());
+        operationHandlers.put(Operation.PURCHASE, new PurchaseOperation());
+        operationHandlers.put(Operation.RETURN, new ReturnOperation());
+        operationHandlers.put(Operation.SUPPLY, new SupplyOperation());
```

Comment:
Avoid using actual implementations in unit tests if possible. Consider using mock objects for 'OperationHandler' implementations to ensure that the test only focuses on the behavior of the 'OperationStrategyImpl' class.